src/parser/esprima_cpp/esprima.cpp

```diff
@@ -4523,10 +4486,19 @@ class Parser {
 
         bool isEndedWithReturnNode = referNode && referNode->type() == ASTNodeType::ReturnStatement;
         if (!isEndedWithReturnNode) {
-            referNode = body->appendChild(builder.createReturnStatementNode(nullptr));
+            Node* returnNode;
 #ifdef ESCARGOT_DEBUGGER
-            referNode->setBreakpointInfo(this->lookahead.start, this->lookahead.lineNumber);
+            if (!referNode) {
+                returnNode = builder.createReturnStatementNode(nullptr);
+            } else {
+                returnNode = builder.createReturnStatementNode(nullptr);
+            }
```

Comment:
What is the difference in these two cases?

Reply:
Thanks, I've forgotten to remove it.